pkg/passive/sources.go

```diff
@@ -34,8 +34,48 @@ import (
 	"github.com/projectdiscovery/subfinder/pkg/subscraping/sources/zoomeye"
 )
 
-// DefaultSources contains the list of sources used by default
+// DefaultSources contains the list of fast sources used by default.
 var DefaultSources = []string{
+	"alienvault",
+	"binaryedge",
+	"bufferover",
+	"certspotter",
+	"certspotterold",
+	"censys",
+	"chaos",
+	"crtsh",
+	"dnsdumpster",
+	"hackertarget",
+	"intelx",
+	"ipv4info",
+	"passivetotal",
+	"securitytrails",
+	"shodan",
+	"spyse",
+	"sublist3r",
+	"threatcrowd",
+	"threatminer",
+	"virustotal",
+}
+
+// DefaultRecursiveSources contains list of default recursive sources
+var DefaultRecursiveSources = []string{
```

Comment:
I think it could be better setting the source as recursive via a property on each source file (e.j: `isRecursive()`) and asking for that property before running instead of using the config file. Any thoughts?

Reply:
Yep, i was thinking about that, marking a source as default, recursive etc using functions and doing stuff based on that. Using interfaces would solve that however this was a quick change since the feature is much needed. We could do a bigger refactor doing many more things like `interfaces`, `tests`, etc.  This will be done in the future.

Reply:
I am working on a local branch to simplify the key management, I will keep this on mind.